### ISSUE TITLE
multiple fastq files

### ISSUE THREAD

**Issue by lfrangeu** [wdecoster/nanocomp#74]
Is it possible to use nanocomp to compar runs if each run contain several fastq.gz files ?

**Comment by wdecoster** [comment]
No, that is currently not supported. There is some code to do something similar with summary files, but that would require changes for the fastq files.

**Comment by lfrangeu** [comment]
Thank you for this quick response. I would really like to be able to do this in a future version. Declare folders instead of files for each run in the command line. I'm not a fan of very large files, and I'll have to concatenate all my fastq.gz to be able to use Nanocomp.


--
Lionel Frangeul
Viruses and RNAi 
Institut Pasteur,
Université Paris Cité, CNRS UMR3569
25 rue du Dr ROUX,
F-75015 Paris, France.

________________________________
De : Wouter De Coster ***@***.***>
Envoyé : jeudi 16 novembre 2023 15:07:47
À : wdecoster/nanocomp
Cc : Lionel FRANGEUL; Author
Objet : Re: [wdecoster/nanocomp] multiple fastq files (Issue #74)


No, that is currently not supported. There is some code to do something similar with summary files, but that would require changes for the fastq files.

**Comment by wdecoster** [comment]
Yes, that makes sense. And you cannot use NanoComp at the summary or BAM/CRAM stage?
